core/arch/aarch64/proc.c

```diff
@@ -112,6 +112,27 @@ proc_num_simd_sse_avx_saved(void)
     return 0;
 }
 
+int
```

Comment:
> of lazy context copying since no AVX-512 instruction are introduced. Instead, we stick to

grammar: instructions

> the process' xsave configuration.

grammar: process's

Reply:
Fixed.